sources/packages/backend/apps/db-migrations/src/migrations/1721687574277-InsertStudentNotificationSupportingUserInformationRequired.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class InsertStudentNotificationSupportingUserInformationRequired1721687574277
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData(
+        "Insert-student-supporting-user-information-required-message.sql",
+        "NotificationMessages",
+      ),
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData(
+        "Rollback-insert-student-supporting-user-information-required-message.sql",
```

Comment:
Rollback is failing. Please rename the file `Delete-insert-student-supporting-user-information-required-message.sql` to `Rollback-insert-student-supporting-user-information-required-message.sql`

![image](https://github.com/user-attachments/assets/6a6b5c9c-cfdd-419a-bc7b-986b224229a1)